index.html

```diff
@@ -1,2 +1,12 @@
 <html>
+    <head>
+        <style>
```

Comment:
Hey buddy can you please put this css in an external file and resumbit your pr.

Reply:
Sure thing